packages/text/.npmignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+test
```

Comment:
`test` directory now renamed to `spec`, so we don't need this line

Reply:
removed it:ok_hand: